blackbox/blackbox_exporter.service

```diff
@@ -0,0 +1,19 @@
+# -*- mode: conf -*-
+
+[Unit]
+Description=Prometheus exporter allowing blackbox probing of endpoints over HTTP, HTTPS, DNS, TCP and ICMP.
+Documentation=https://github.com/prometheus/blackbox_exporter
+After=network.target
+
+[Service]
+PermissionsStartOnly=true
+User=ccp_monitoring
+EnvironmentFile=/etc/sysconfig/blackbox_exporter
+ExecStart=
+ExecStart=/usr/bin/blackbox_exporter $OPT
+ExecReload=/usr/bin/kill -HUP $MAINPID
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+
```

Comment:
Please pass this service file to build team and get a blackbox rpm.

Reply:
Done. This file will be removed in followup commit